src/style.css

```diff
@@ -0,0 +1,128 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  background-color: #eef;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 0 17px;
```

Comment:
you'd rather add vertical paddings here, top padding for image + bottom margin for button — not OK